---
content_type: page
description: This section contains details on the assignment for Module 1.
draft: false
learning_resource_types:
- Assignments
ocw_type: CourseSection
parent_title: Assignments
parent_type: CourseSection
parent_uid: 1b88a657-74a1-a702-79bf-92c805387c4a
title: 'Module 1 Assignment: Genome Engineering Portfolio'
uid: dc3a75af-65bf-67a8-6b22-50d2a25be03a
video_files:
  video_thumbnail_file: null
video_metadata:
  youtube_id: null
---
Portfolio components:

## Part 1: Rebuttal to Editorial

This will be written as a homework assignment associated with lab module 1.4 (Examine candidate clones), exchanged with your lab partner for peer review, then submitted to the teaching faculty as part of your portfolio. **This portion of the assignment accounts for up to 15% of your grade**.

Essay 1: Choose **one** of the following quotes to address. Both come from Andrew Pollack in the New York Times, Tuesday, Jan 17, 2006, {{% resource_link "ecaa9cf4-3065-4f97-b9d8-d6f6ef5ad6ea" "Custom-Made Microbes, at Your Service" %}} which quotes Professor Arnold of Caltech as saying:

- "(Synthetic Biology) has a catchy new name, but anybody over 40 will recognize it as good old genetic engineering applied to more complex problems."

and

- "There is no such thing as a standard component, because even a standard component works differently depending on the environment. The expectation that you can type in a sequence and can predict what a circuit will do is far from reality and always will be."

Essay 2: "Editorial: Meaning of Life." In *Nature* (2007) 447: 1031-1032:

- "it would be a service…to dismiss the idea that life is a precise scientific concept"

## Part 2: M13.1 Redesign Description and Parts

Submit your annotated sequence as well as a human-readable text to describe your refactoring. What do you need to do exactly? First, choose a section of the M13.1 scaffold in which you will do your refactoring work (you should choose what you want to refactor based on what seems interesting or important to you). Do not try to refactor all of M13. Only one section between two unique restriction endonuclease sites. Second, design the sequence of DNA for your refactored section of the M13 genome (**Note**: your DNA sequence must start and end with two of the available unique restriction endonuclease sites across the M13 genome). Third, provide the annotated sequence for your refactored section (follow the color scheme suggested, or a better, clearly defined scheme of your own). Don't do anything fancy. A rich text file, word document, or powerpoint file should be good enough. Fourth, provide a one paragraph description of what the purpose or goal of your refactored section is (e.g., what will it test? or, what new function will it provide?). Also, include a cost estimate for your refactored section ($1.35 per bp) and tell us whether or not you think we should have the section synthesized (GO or NO GO). We will quickly check your designs (regardless of GO/NO GO), follow-up w/ any questions, and then order them for DNA synthesis (if you or we say GO). Because we are trying to do this v. quickly this term, it is very important that your clearly annotate your designed sequence (so that we can check it), and provide a good written paragraph describing what your design goals are (so that we can more readily understand what you are hoping to do). **This portion of the assignment accounts for up to 40% of your grade**.

## Redesign Ideas from 20.109 (S07)

{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
GENE
{{< thclose >}}{{< thopen >}}
IDEAS
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
II
{{< tdclose >}}{{< tdopen >}}

Make p2 sensitive to and perhaps degraded by any one of various stimuli (e.g. heat, light, pH, chemical input) so replication can be regulated

Encode so that p2 can be switched on and off based on the environment which it's in, for example: stops replication of phage genome when in a certain concentration of Ca{{< sup "2+" >}} enters the cell

Make p2 require a cofactor that must be added before replication of the phage begins

Make p2 count the number of times it nicks DNA

Modify p2 in a way that helps p5 sequester the + strands more effectively, perhaps making p5 work more efficiently by reducing competition

Modify such that it not only nicks the double stranded form of the genome to initiate replication of the + strand, but also nicks the - strand to impede the formation of dsDNA (this would also help p5)

Separate g2 and g10 by inserting entire sequence of g10 after transcription end of g2/g10 (essentially repeating g10 twice in succession), so that the two genes are independent

Alter the gene to make it more active and thus replicate DNA more frequently — see how increased DNA production affects phage growth

Modify so it can nick foreign DNA (e.g. the E. coli's) and the phage can replicate and package a portion of the host DNA

Remove p2 and p5 to see if the bacteria still make the phage gene products without replicating the phage DNA

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
X
{{< tdclose >}}{{< tdopen >}}

Make p10 sensitive to a different stimulus than p2 to again regulate replication

Perhaps a dual control mechanism for p2 and p10 expression

Increase the number of p10 so that the phage can produce more double strands

Modify such that the + strands of DNA are not soley dependent on the presence of p10. This modification works together with our modification of p2

Put a tag on p10 to see what it binds at various parts of replication. This will help elucidate how it controls the amount of double stranded M13 genomes

Modify to add another level of regulation for phage propagation. This, coupled with control of II, could allow complex control of the life cycle behavior of the virus

Make pX more active so that more + strands will accumulate, allowing the host cell to produce even more phages

Extract from gene II

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
V
{{< tdclose >}}{{< tdopen >}}

Add a tag different from p8 (e.g. RFP) to determine what stage of the phage life cycle it is in or to monitor levels of p5-ssDNA complex

Alter interaction with p9/p7 so that a limited number of ssDNA may be surrounded by p8 at a time

Make it sensitive to mechanism that allows for assay of DNA amount and location, change its assembly mechanism to influence phage size

Modify protein so that similar proteins, other than p8 can bind to the surface

Remove overlap of p5 gene with start codon of gene for p7. An activation site could be added in prior to start codon in gene for p7 that could be used to allow only limited amounts of pV ( and pVII) to be expressed. This could give control over the the quantity of fully assembled phage to be produced by the host

Modify such that p5 can sequester the + stranded DNA more effectively so that there is less competition with the formation of double stranded

Vary the activity of the protein and thus the competition between dsDNA formation and the sequestering of ssDNA- compare the results to find the optimum level of phage production possible

Modify expression so that we can control the amount of time the virus DNA spends inside the host (as opposed to actively being packaged and spreading to other bacteria)

Allow it to sequester double stranded DNA also, then it can be used as a vector for infecting cells with desired DNA fragments

Add more DNA binding sites to see if more DNA can be packaged into phage

Add some base pairs between V and VII to allow for a restriction site

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
VII
{{< tdclose >}}{{< tdopen >}}

Alter gene so protein adopts different/more flexible conformation. A change in conformation might expand the different residues that can be attatched to its N-terminal portion

Change the way p7 interacts with p9

Modify to increase rate host will shed phage, i.e. decrease the phage-host interaction time

Modify the last few codons to remove overlap with the gene encoding p9

Add sequence to the 5' end of gene so protein could build nanowires or long filaments or other useful materials

Delete to learn more about its function

Increase it's expression to learn more about its function

Tag protein to monitor interaction with p5/DNA complex

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
IX
{{< tdclose >}}{{< tdopen >}}

Modify p9 to bind to p3 to create long filaments of phage lined up end to end

Modify to express different reactive chains on the phage surface

Change the function so that p9 will now lyse the bacteria

Modify to make the phage secretion occur at a faster rate so that interaction time with the host is reduced

Modify beginning and end sequences so that g9 does not overlap with g7 and g8

Modify the p9 so that it can bind to bacterial surface proteins (the way p3 does) — see if this allows the phage to interact with other bacteria (now that both ends of the phage can bind and perhaps bridge the two bacterial cells)

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
VIII
{{< tdclose >}}{{< tdopen >}}

Add epitope or other tags, e.g. x-ray sensitive, UV sensitive, flourescent

Alter gene so p8 has an affinity for certain residues or salts. This can vastly increase the function of m13 as a whole. It can be used to transport different things into bacteria

Change p8 interactions with p5 to regulate size of phage or influence shape of phage by changing how it assembles into a coat

Alter gene so fewer copies of p8 fit on phage coat allowing a more flexible packing structure

Insert sequence between genes for p5 and p8 to isolate production of these proteins and control their expression levels via activation or repression sites

Change amino acid sequence to allow p8 (and thus the entire virus) to bind to certain materials, like metals

Add a small protein to the gene for p8 to amplify because p8 is synthesized so many times

uncoupled gene for p8 from the gene for p9

{{< tdclose >}}{{< trclose >}}{{< tropen >}}{{< tdopen >}}
III
{{< tdclose >}}{{< tdopen >}}

Add epitope or other tags, e.g. X-ray sensitive, UV sensitive, flourescent

Modify portion of p3 that normally interacts with TolA on bacterial pilus to vary phage selectivity to different or only certain bacteria

Make p3 bigger so that the phage can bind to objects easier and so that they might bind to bigger objects

Selective degredation of p3 to control the maximum amount of DNA allowable on the phage sequence

Modify to monitor the timing of the phage escape from host

Add sequence to make the tail protein longer

Make series of changes to explore the mechanism by which p3 enters and exits the cell. For example: exchange charged amino acids for neutral, acidic for neutral or basic, etc.

Delay the time at which the p3/p6 cap is added by making p3 expression a function of environmental cues such as ionic strength or pH. Could also study the effect this would have on the infection process since p3 is also the protein which binds to the TolA protein on the bacterial pilus

Change the GTG to ATG start

{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

## Part 3: Data Summary for p3-Modifications You Performed in Lab

Consult the {{% resource_link "fc19e690-0ca7-af8b-d48d-3a5a9e329f01" "Guidelines for writing a lab report" %}}. Additionally, many of the "for next time" assignments can get you started on this part of the portfolio. Including but not limited to:

- Table: Oligonucleotide design, sequence consequences for phage when inserted and sequence data
- Table: Ligation results
- Figure: Agarose gel examining candidate clones
- Figure: Western results
- Table: Plaque assay
- Short paragraph for each table and figure describing and interpreting what's shown
- One or two sentence summary of your experimental results
- One or two sentence proposal for what you'd do next if we had one more month to spend on this project

**This portion of the assignment accounts for up to 20% of your grade.**

## Part 4: Mini-business Plan for the Registry of Standard Biological Parts

Put yourself 5 years in the future and imagine that the Registry is floundering. Though the number of useful parts has grown through the hard work and dedication of its volunteer workforce in the iGEM program, there is a notable lack of standards:

- around the parts themselves (some work always, some in rare conditions, some not at all)
- around the assembly process (alternative biobricks and registries have gained popularity)
- and around documentation for the parts (some have great spec sheets and some have nothing)

Decide that you will direct the Registry into a manufacturing, service, high tech, or retail business and then devise a plan to grow and stabilize that business. **This portion of the assignment accounts for up to 25% of your grade.**

In no more than three pages provide a business plan that includes:

### 1\. An Executive Summary

- In 250 words or fewer, explain:   
      
    - What is your product
    - Who are your customers
    - What the future holds for the registry in particular and synthetic biololgy more generally
    - What you see as the key to success
- This summary should sound enthusiastic, professional and be more readable than most "mission statements."
- Consider writing this section after you've written the rest of the plan.
- Describe what the Registry is, including products, services, customers, ownership, history, location, facilities.
- Include strengths and core competencies of the Registry.
- Segue into the next section by mentioning the significant challenges faced in the near and long term.
- This section should be no longer than 2 paragraphs.

### 2\. Summary of the Current Registry

- Describe what the Registry is, including products, services, customers, ownership, history, location, facilities.
- Include strengths and core competencies of the Registry.
- Segue into the next section by mentioning the significant challenges faced in the near and long term.
- This section should be no longer than 2 paragraphs.

### 3\. Market Analysis

Dedicate one paragraph to a description of the market. You might consider including information like:

- Who makes up your market?
- What is it's size now? how fast is it growing? how do you know?
- What percentage of the market do you expect the Registry to have now and 5 years from now?
- How could changes in technology, government, and the economy affect your business?

### 4\. Business Plan

Specify your strategy for continued growth of the Registry. The emphasis of this section will differ depending on the kind of business model you have chosen (retail, manufacturing, service or high tech)

Here are some questions you might consider as you formulate your business plan:

- How will you promote the use of the Registry?
- How will you advertise?
- How will you price your product/services?
- Where will you locate the Registry (or BioBrick franchises) and how you will distribute parts/services?
- How you will keep the Registry competitive?
- How/if you will protect intellectual property while also promoting sharing and community?
- Does your plan emphasize increased production, diversification, or eventual sale of franchises?
- How long will your strategy take to be partially or fully realized?
- Are there start-up costs associated with your business model? How much and where will the capital come from?
- Will your registry require insurance coverage or litigation insurance?
- Are there trademarks, copyrights, or patents (pending, existing, or purchased) considerations?
- How many and what kind (skilled, unskilled, and professional) of employees to you anticipate?
- Where will you recruit employees?
- Will top notch employees advance? To what?
- How will you training employees?
- What kind of inventory will you keep: raw materials, supplies, finished goods?
- Will there be seasonal fluctuations to demand for parts?
- Will you need lead-time for ordering?
- Do you expect shortages or delivery problems?
- Are supply costs steady? Reliable?
- Will you sell parts on credit?
- How will you set prices?
- What kind of guarantees and privacy protects will you offer?

This section has no defined length or format but should end on an enthusiastic note that might lead some venture capital firm or a funding agency to stay interested.